### PR TITLE
Admin forms

### PR DESCRIPTION
**Pull request by noirbizarre**
This pull request is proposal for an admin improvement:
- full page forms
- new page menu
- display some missing data 

screenshot-admin-dataset-new

screenshot-www data dev 2016-03-24 11-52-19

screenshot-admin-dataset-form

Pages to migrate:

- [x] Dataset page and form
- [x] Organisation page and form
- [x] Reuse page and form
- [x] Harvester view and form
- [x] Topic page and form
- [x] Profile page and forms
- [x] Post page and form

### CODE REVIEW

**Comment by davidbgk** [comment]
Looks good so far! I think we need to iterate on buttons placement but let's do that IRL next week ;)

**Comment by noirbizarre** [comment]
I agree, there's still somm rooms for improvement on the header.
The button placement is not optimal.
I'd like to put it on the top right and move le site on site elsewhere

**Comment by noirbizarre** [comment]
Proposal:

screenshot-www data dev 2016-03-24 15-01-10

**Comment by davidbgk** [comment]
Better! Can we unstack the button on the right to a bar without any dropdown?

**Comment by noirbizarre** [comment]
I think this is too large. When there is only one action, there is no dropdown.
right now, extra actions are:
- delete
- transfert
- admin actions (badges...)

File: js/views/dataset.vue [hunk 1]
```diff
@@ -131,13 +156,17 @@ export default {
                 color: 'purple',
                 target: '#trafic'
             }];
+        },
+        territories_labels() {
+            if (this.geojson && this.geojson.features) {
+                return this.geojson.features.map(function(feature) {
```
Comment (davidbgk) [inline js/views/dataset.vue]: `feature => feature.properties.name` ;)

Comment (noirbizarre) [inline js/views/dataset.vue]: Oh, a shortcut, I'll try !

**Comment by noirbizarre** [comment]
Added scroll-to submenu (only one active, scroll to a given section):

screenshot-www data dev 2016-03-29 17-15-11

**Comment by davidbgk** [comment]
r+

**Pull request merged**